src/main/java/uk/ac/cam/cl/dtg/util/email/MailJetApiClientWrapper.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2021 James Sharkey
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ * 		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.cam.cl.dtg.util.email;
+
+import com.google.inject.Inject;
+import com.mailjet.client.ClientOptions;
+import com.mailjet.client.MailjetClient;
+import com.mailjet.client.MailjetRequest;
+import com.mailjet.client.MailjetResponse;
+import com.mailjet.client.errors.MailjetClientRequestException;
+import com.mailjet.client.errors.MailjetException;
+import com.mailjet.client.resource.Contact;
+import com.mailjet.client.resource.ContactManagecontactslists;
+import com.mailjet.client.resource.Contactdata;
+import com.mailjet.client.resource.Contacts;
+import com.mailjet.client.resource.ContactslistImportList;
+import org.apache.commons.lang3.Validate;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.ac.cam.cl.dtg.util.PropertiesLoader;
+
+import javax.ws.rs.core.Response;
+
+import static uk.ac.cam.cl.dtg.segue.api.Constants.*;
+
+public class MailJetApiClientWrapper {
+
+    private static final Logger log = LoggerFactory.getLogger(MailJetApiClientWrapper.class);
+    private final MailjetClient mailjetClient;
+    private final String newsListId;
+    private final String eventsListId;
+
+    /**
+     *  Wrapper for MailjetClient class.
+     * @param properties - needed to get the MailJet keys and list IDs
+     */
+    @Inject
+    public MailJetApiClientWrapper(final PropertiesLoader properties) {
```

Comment:
Rather than passing in the whole properties loader can we use Guice to magically create one of these with clear dependencies in the constructor params?